m/moss/pkg/moss-benchmark-time-to-prime-asset-store-vfs-cache.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
```

Comment:
is there some way to make this conditional or disabled by default ? Maybe some command line presence in the kernel? Just don't want to negatively impact low n-core systems

Reply:
Hopefully resolved by newest commit.